### DISCUSSION TITLE
Cloudlog Documentation

### DISCUSSION THREAD

**Discussion by magicbug** [magicbug/Cloudlog#741] [General]
There's always some ongoing documentation projects going on however normally tucked away in private conversations so time to make it public to let people lend a hand, as writing good docs isn't my strong point.

If anyone knows of areas that need to be improved then let's try to figure it out!

- [x] My Sig and My Sig Info in Station Profile
- [x] Sig and Sig Info in Logging
- [ ] Station Profile docs need improved
- [ ] Logging area could do with being explained with information about the different fields and whats expected.

**Comment by ei8fdb** [comment]
I was going to ask this exact question. But instead, I'll ask what areas of documentation do you think need work?

I'm still working my way through it all, so don't have a good sense yet. One area that's confusing to me, so far, it getting LOTW working, and what actually gets updated.

Of your list above, which are still in need of documentation?

**Reply by magicbug** [reply]
Everything around Station Logbooks & Locations needs explaining a lot better, which I've struggled to explain well although its important

As for LoTW it's probably needing some improvement its a fairly complex part of Cloudlog

**Reply by ei8fdb** [reply]
OK, I'll try to start off with Station Logbooks and Locations. Do you have a documentation style guide, or particular approach to writing docs? This is different from code style guide.

[Gitlab's](https://www.writethedocs.org/guide/writing/style-guides/) has some good examples (though it is probably overkill for Cloudlog), and then Write The Docs has a nice [example list](https://www.writethedocs.org/guide/writing/style-guides/) of style guides from projects/companies.

If not, I'll write it how I'd do it and tell me what you think.
